__tests__/__unit__/uss/ussNodeActions.unit.test.ts

```diff
@@ -331,38 +331,6 @@ describe("ussNodeActions", () => {
         });
     });
 
-    describe("deleteUSSNode", () => {
```

Comment:
so we are not testing delete USS NOde?

Reply:
Yes, in _ZoweUSSNode.unit.tests_, since the function is called on the node: `node.deleteUSSNode()`

If it were called like `deleteUSSNode(node)`, it would still be in the _ussNodeActions.unit.test.ts_ file.